packages/client/README.md

```diff
@@ -550,137 +664,100 @@ const dataUnion = await client.deployDataUnion({
 Data Union deployment uses the [CREATE2 opcode](https://eips.ethereum.org/EIPS/eip-1014) which means
 a Data Union deployed by a particular address with particular "name" will have a predictable address.
 
-## Utility functions
+### Utility functions
+The static function `StreamrClient.generateEthereumAccount()` generates a new Ethereum private key and returns an object with fields `address` and `privateKey`. Note that this private key can be used to authenticate to the Streamr API by passing it in the authentication options, as described earlier in this document.
```

Comment:
This could be a bit shorter explanation as we've already introduced this earlier?

Reply:
I'd remove lines 674-679. This chapter is about the utility functions, we may assume that users know if they need them? (We've linked to this section)

Reply:
Sorry, which lines are you suggesting to remove? The utility method text has changed since this comment:
https://github.com/streamr-dev/network-monorepo/blob/98f9488b48d71a08fc67b2001f2ac453ab418e77/packages/client/README.md#utility-functions

Reply:
These:
```
Note that this private key can be used to authenticate to the Streamr API by passing it in the authentication options, as described earlier in this document:
...
const streamr = new StreamrClient({
    auth: { privateKey }
})
```